src/common.c

```diff
@@ -2272,3 +2272,16 @@ size_t ofi_vrb_speed(uint8_t speed, uint8_t width)
 
 	return width_val * speed_val;
 }
+
+#define MAX_LOG_PREFIX_LEN 256
+static char log_prefix[MAX_LOG_PREFIX_LEN];
+
+void ofi_set_log_prefix(const char *prefix)
+{
+	strncpy(log_prefix, prefix, MAX_LOG_PREFIX_LEN);
+}
```

Comment:
I'm not a fan if adding accessor functions just to set/read a simple variable.  In this case, we end up replacing a pointer assignment with a string copy function that may not produce a null terminated string.

Reply:
Would you prefer simply moving the variable `log_prefix` from `src/log.c` to `src/common.c`?

Reply:
Relative to this change, yes, but see my comment in the related issue.  If we build rxm as a DL provider, we won't get the expected results. Maybe that's okay, as we're at least the same or better than where things stand now.

Reply:
Made that change in new revision, verified that fixed the EFA dso build. Not sure about rxm though.